test/bigtest/tests/layer-po-line-test.js

```diff
@@ -0,0 +1,67 @@
+import { beforeEach, describe, it } from '@bigtest/mocha';
```

Comment:
`layer-po-line-test.js` - we don't test components. treat it like e2e tests

Reply:
as I don't understand exactly how to calculate test coverage I decided to create new test file. After your comment I moved my test cases into existing file

Reply:
you don't need to calculate test coverage, it's done on SonarCloud. https://github.com/folio-org/ui-orders/pull/343/checks